src/InvokePlaster.ps1

```diff
@@ -805,7 +805,8 @@ function Invoke-Plaster {
                     if ($? -and $oldModuleManifest) {
                         $props = 'Guid', 'Description', 'DefaultCommandPrefix', 'RootModule', 'AliasesToExport',
                                  'CmdletsToExport', 'DscResourcesToExport', 'VariablesToExport',
-                                 'FormatsToProcess', 'TypesToProcess', 'ScriptsToProcess'
+                                 'FormatsToProcess', 'TypesToProcess', 'ScriptsToProcess', 'Tags', 'Author',
```

Comment:
I don't mind adding these but I worry a bit about trying to stay up-to-date with new module manifest fields.  The original idea was to provide support for just the commonly modified fields provided as part of a template.  But what is "commonly modified fields" is definitely debatable.  :-)